### ISSUE TITLE
Switch back to FamilySearch/GEDCOM-registries

### ISSUE THREAD

**Issue by dthaler** [ArmidaleSoftware/gedcom7#117]
once https://github.com/FamilySearch/GEDCOM-registries/pull/11 is merged,
it will generate another PR in FamilySearch/GEDCOM-registries, similar to
https://github.com/dthaler/GEDCOM-registries/pull/2

Merging that generated PR will unblock switching back.
